js/src/index.parity.js

```diff
@@ -52,12 +53,34 @@ if (process.env.NODE_ENV === 'development') {
 }
 */
 
-const AUTH_HASH = '#/auth?';
-
+const TOKEN_QS = '?token=';
+const hashIndex = window.location.hash
```

Comment:
Perhaps it's worth to extract this into a utility function so that we can test it?
```
function redirect(protocol, hostname, hash, search, localStore) {
 ... 
}
```

Reply:
Makes sense.